pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Common.java

```diff
@@ -677,31 +672,16 @@ private static String getPrimitiveWrapperCompareToGeneration(Field f) {
                     """;
 
 
-		final String compareStatement;
-		switch (f.messageType()) {
-			case "StringValue" ->
-					compareStatement =
-							"$fieldName.compareTo(thatObj.$fieldName)";
-			case "BoolValue" ->
-					compareStatement =
-							"java.lang.Boolean.compare($fieldName, thatObj.$fieldName)";
-			case "Int32Value", "UInt32Value" ->
-					compareStatement =
-							"java.lang.Integer.compare($fieldName, thatObj.$fieldName)";
-			case "Int64Value", "UInt64Value" ->
-					compareStatement =
-							"java.lang.Long.compare($fieldName, thatObj.$fieldName)";
-			case "FloatValue" ->
-					compareStatement =
-							"java.lang.Float.compare($fieldName, thatObj.$fieldName)";
-			case "DoubleValue" ->
-					compareStatement =
-							"java.lang.Double.compare($fieldName, thatObj.$fieldName)";
-			case "BytesValue" ->
-					compareStatement =
-							"(int)($fieldName.length() - thatObj.$fieldName.length())";
+		final String compareStatement = switch (f.messageType()) {
+			case "StringValue" -> "$fieldName.compareTo(thatObj.$fieldName)";
+			case "BoolValue" -> "java.lang.Boolean.compare($fieldName, thatObj.$fieldName)";
+			case "Int32Value", "UInt32Value" -> "java.lang.Integer.compare($fieldName, thatObj.$fieldName)";
+			case "Int64Value", "UInt64Value" -> "java.lang.Long.compare($fieldName, thatObj.$fieldName)";
+			case "FloatValue" -> "java.lang.Float.compare($fieldName, thatObj.$fieldName)";
+			case "DoubleValue" -> "java.lang.Double.compare($fieldName, thatObj.$fieldName)";
+			case "BytesValue" -> "(int)($fieldName.length() - thatObj.$fieldName.length())";
```

Comment:
nit: This might be clearer as `java.lang.Byte.compareUnsigned($fieldName, thatObj.$fieldName)`

Reply:
It turned out that `BytesValue` translates to `com.hedera.pbj.runtime.io.buffer.Bytes` which is comparable now, so I refactored it to this:
```
case "StringValue", "BytesValue" -> "$fieldName.compareTo(thatObj.$fieldName)";
```